share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -204,19 +204,21 @@ endfunction()
 # BOARD <board>:             Use <board> for application build instead user defined BOARD.
 # BOARD_REVISION <revision>: Use <revision> of <board> for application (only valid if
 #                            <board> is also supplied).
-# APP_TYPE <MAIN|BOOTLOADER>: Application type.
+# APP_TYPE <MAIN|BOOTLOADER|UPDATEABLE>: Application type.
 #                             MAIN indicates this application is the main application
 #                             and where user defined settings should be passed on as-is
 #                             except for multi image build flags.
 #                             For example, -DCONF_FILES=<files> will be passed on to the
 #                             MAIN_APP unmodified.
 #                             BOOTLOADER indicates this app is a bootloader
+#                             UPDATEABLE indicates this app is updateable and the bootloader
+#                             settings should be passed on.
```

Comment:
no that is not really what this config item is for, this is for an image type. Anyhow you can pass these settings on in sdk-nrf without touching sdk-zephyr, see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/CMakeLists.txt#L344

Reply:
File moved: https://github.com/nrfconnect/sdk-nrf/pull/22401